docs/nydusd.md

```diff
@@ -239,6 +239,45 @@ Once the configuration is loaded successfully on nydusd starting, we will see th
 INFO [storage/src/backend/connection.rs:136] backend config: CommonConfig { proxy: ProxyConfig { url: "http://p2p-proxy:65001", ping_url: "http://p2p-proxy:40901/server/ping", fallback: true, check_interval: 5 }, timeout: 5, connect_timeout: 5, retry_limit: 0 }
 ```
 
+##### Enable Mirrors for Storage Backend
+
+Add `device.backend.config.mirrors` field to enable mirrors for storage backend. The mirror can be a P2P distribution server (such as [Dragonfly](https://d7y.io/)) or registry. If the request to mirror server failed, it will fall back to the original registry.
+Currently, only registry supports mirror, other backends will support it soon.
```

Comment:
The OSS backend is also supported? And better to tell that the `mirrors` field conflicts with `proxy` field.

Reply:
Not implemented now, removed.
Yes, added this conflicting note.